packages/notionable/src/global/account/communities.ts

```diff
@@ -97,3 +97,7 @@ export function checkSanctionedAddress(account: string) {
   );
   return sanctionList['isSanctioned'](account) as Promise<boolean>;
 }
+
+export function checkNewUserAddress(account: string) {
+  return newUserAddresses.includes(account);
```

Comment:
Make sure that all the account address is lower case here and all the addresses in the json are lower case as well.